content/References/IPv6.md

```diff
@@ -0,0 +1,247 @@
+---
+title: "Understanding IPv6"
+description: "Provides information general information on IPv6 and implementation guidance."
+weight: 20
+tags:
+- network
+- interfaces
+- ipv6
+---
+
+## What is IPv6?
+IPv6 is the next generation Internet protocol replacing the current IPv4 protocol.
+IPv4 is running out of numbers due to the ever increasing number devices world wide wanting to connect.
+The rise of the Internet of Things (IoT), where smart sensors in homes, motor vehicles, and businesses all want or need to connect to the Internet to allow data collection and sharing for analytics, device maintenance and monitoring, increased safety and productivity, and overall improvement in consumer and customer experiences is a source of an ever-increasing demand for IP addresses.
+Because of the length and configuration of IPv6 addresses, there is little risk of this protocol running out of numbers any time soon.
+
+### IPv6 Addresses
+An IPv6 address is a 128-bit alphanumeric address organized into eight groups of 16 bits with each group expressed as four-digit hexadecimal numbers separated by colons.
+An example of an IPv6 address is *fe80:cd00:0000:0cde:1257:0000:211e:729c*.
+
+IPv6 addresses are split into two parts: a network (routing prefix) and a node component(interface ID).
+
+The network component (commonly 64 bits) is the part of the address and is used for routing.
+This component can be split into a block for global network addresses and a block network administrators can use for subnets on an internal network.
+
+The node component is the second part of the IPv6 address.
+If using auto-configuration, this part of the address can be derived from a physical or MAC address, using the 64-bit extended unique identifier (EUI-64) format defined by the Institute of Electrical and Electronics Engineers (IEEE).
+This format designed to provide the ability to provide unique addresses to any electronic device equipped with sensors and connect them for data sharing.
+
+The IPv6 total address can be shortened. The addressing scheme allows the omission of any leading zero as well as any sequences consisting of only zeros.
+Using the example above, a shorted address can look like *fe80:cd00:0:cde:1257:0:211e:729c*.
+
+The specific layout of IPv6 addresses can vary depending on its format. 
+It has three basic parts: routing prefix, subnet ID, and the interface ID.
+The routing prefix and subnet ID can represent either global or site specific information.
+The interface ID can be manually or automatically configured.
+
+## What are the Benefits of IPv6?
+IPv6 eliminates the need for:
+
+* {{< expand "Network address translation (NAT)" "v" >}}
```

Comment:
Looking at the rendered local build, I don't think it's necessary to use both bullets and the expand element (which has it's own indent and carrot). I'd remove the bullets from these and the next list

Reply:
I tried it with and without the bullets. I'm partial to with but have no issue removing them.